### PR TITLE
Add the note and minor corrections in the main README.md document

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Added a note about a similar document in the `kyma` repository and the need to update it when the examples table is changed.
- Updated the README.md for the EC example to represent the current product name.
- Reviewed the README.md for the EC example and introduced minor fixes.
- Modified the EC-related diagram accordingly.

**Related issue(s)**
Relates to #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,17 +19,19 @@ The examples project provides a central repository to showcase and illustrate fe
 
 The summary of the documentation in the `examples` repository lists all available examples organized by the feature or concept they showcase. This structure provides a quick overview and easy navigation.
 
+>**NOTE:** The table with examples is also available under `docs` in the `kyma` repository. Whenever you update the table, modify the [Overview](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/007-overview-examples.md) document accordingly.
```
Comment (derberg) [inline README.md]: Please make it a hidden html comment that is visible only to the editors

like

Comment (kazydek) [reply README.md]: Sure

File: README.md [hunk 2]
```diff
@@ -19,17 +19,19 @@ The examples project provides a central repository to showcase and illustrate fe
 
 The summary of the documentation in the `examples` repository lists all available examples organized by the feature or concept they showcase. This structure provides a quick overview and easy navigation.
 
+<!-- NOTE: The table with examples is also available in the kyma repository. Whenever you update the table, modify this Overview document accordingly: https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/007-overview-examples.md. -->
```
[the remark this answers is not in the document]
Comment (kazydek) [reply README.md]: Actually, I didn't add it intentionally as it is a hidden comment that does not render in preview thus markdown formatting is not required here.

File: call-ec/README.md [hunk 3]
```diff
@@ -88,45 +88,48 @@ kubectl apply -f ./expose-ec-function.yml
...
 ```bash
 curl -i -G https://calculate-promotion.{YOUR.CLUSTER.DOMAIN} -d "user-id={customer_id}" -d "threshold=1000" -H "occ-token: {EC_access_token}" -H "Authorization: Bearer {EC_ID_token}"
 ```
 
 ### Use the UI
 
-To access the UI, go to `http://localhost:3000` and authenticate in Enterprise Commerce. After you authenticate, the system redirects you back to the Web UI application.
-Find the access token and the ID token issued for your application by Enterprise Commerce in the **Authentication** section. Expand **The Function** section to call the function and check the results of the operation.
+To access the UI, go to `http://localhost:3000` and authenticate in SAP Commerce. After you authenticate, the system redirects you back to the Web UI application.
+Find the access token and the ID token issued for your application by SAP Commerce in the **Authentication** section. Expand **The Function** section to call the Function and check the results of the operation.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply call-ec/README.md]: My bad. I will change that.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kazydek) [reply README.md]: Let's change it to `"kyma" repository`.

**Pull request merged**